src/pages/home/filter.ts

```diff
@@ -30,6 +30,7 @@ export interface HomeMsgFilter {
   label: string;
   filter: Filter;
   isValidEvent?: (event: Event) => boolean;
+  authors?: string[];
```

Comment:
why add `authors` here?

Reply:
Oh, this is an incorrect change. I have already removed it.